### PR TITLE
added favicon image in the title

### PR DESCRIPTION
**Pull request by harshita-2003**
Favicon image increased the overall personability of the website in the top section

### CODE REVIEW

**Review by ydv-ankit — CHANGES_REQUESTED** [review]
looks nice but should include favicon in every html file for smooth integration of icon in whole website

File: index.html [hunk 1]
```diff
@@ -10,6 +10,8 @@
 Smartphone Compatible web template, free webdesigns for Nokia, Samsung, LG, SonyEricsson, Motorola web design" />
 <script type="application/x-javascript"> addEventListener("load", function() { setTimeout(hideURLbar, 0); }, false);
 		function hideURLbar(){ window.scrollTo(0,1); } </script>
+<!--favicon icon-->
+<link rel="icon" href="images/restaurant.png">
```
Comment (ydv-ankit) [inline index.html:14]: correct

**Comment by harshita-2003** [comment]
for the smooth integration added favicon to the whole website

**Review by ydv-ankit — APPROVED** [review]
Perfect

**Comment by harshita-2003** [comment]
do acknowledge and merge it

**Review by Anupkjha2601 — APPROVED** [review]

**Pull request merged**